packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -20,20 +20,21 @@ const RootStack: React.FC = () => {
   // remove connection on mobile verifier proofs if proof is rejected
   // regardless of if it has been opened
   const declinedProofs = useProofByState([ProofState.Declined, ProofState.Abandoned])
+  const [onboardingComplete, setOnboardingComplete] = useState(false)
 
   useDeepLinks()
 
-  const isAuthenticated = useMemo(
-    () => store.authentication.didAuthenticate && store.onboarding.postAuthScreens.length === 0,
-    [store.authentication.didAuthenticate, store.onboarding.postAuthScreens]
-  )
+  const isOnboardingComplete = useMemo(() => onboardingComplete, [onboardingComplete])
```

Comment:
I don't think this `useMemo` will have any effect. I think primitive values already only cause re-renders when they change. If they are reassigned to the same value they don't cause re-renders (afaik)

Reply:
Fixed.